app/src/main/java/org/wikipedia/talk/TalkTopicHolder.kt

```diff
@@ -44,20 +46,33 @@ class TalkTopicHolder internal constructor(
 
         binding.topicTitleText.text = titleStr.ifEmpty { context.getString(R.string.talk_no_subject) }
         binding.topicTitleText.visibility = View.VISIBLE
-        binding.topicSubtitleText.visibility = View.GONE
         binding.topicTitleText.typeface = if (seen) Typeface.SANS_SERIF else unreadTypeface
         binding.topicTitleText.setTextColor(ResourceUtil.getThemedColor(context, if (seen) android.R.attr.textColorTertiary else R.attr.material_theme_primary_color))
+
+        val allReplies = threadItem.allReplies
```

Comment:
Could there be a case where `allReplies` is empty? Should we handle such a case?